### PR TITLE
Skip tests on IE because this plugin is not supported by this browser.

### PR DESCRIPTION
**Pull request by antoniopol06**

### CODE REVIEW

**Comment by ipeychev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: test/plugins/test/dragresize.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                Utils.createCKEditor.call(this, done, {extraPlugins: 'ae_dragresize', imageScaleResize: 'scale'});
+            });
+
+            after(Utils.destroyCKEditor);
+
+            beforeEach(Utils.beforeEach);
+
+            afterEach(Utils.afterEach);
+
+            it('The image, that is selected, has 4 handlers', function() {
```
Comment (ipeychev) [inline test/plugins/test/dragresize.js]: The sentence must start with the word 'should', for example:
'should add 4 handlers to a selected image'

File: test/plugins/test/dragresize.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                Utils.createCKEditor.call(this, done, {extraPlugins: 'ae_dragresize'});
+            });
+
+            after(Utils.destroyCKEditor);
+
+            beforeEach(Utils.beforeEach);
+
+            afterEach(Utils.afterEach);
+
+            it('The image, that is selected, has all resize handlers', function() {
```
Comment (ipeychev) [inline test/plugins/test/dragresize.js]: The combination of `describe` and `it` should make sense. Here is what the developer, who is reading the tests will get now:

`imageScaleResize default imageScaleResize "both" The image, that is selected, has all resize handlers`


Please look [here](https://mochajs.org/#getting-started) for an example how to use describe and it

File: test/plugins/test/dragresize.js [hunk 3]
```diff
@@ -3,6 +3,12 @@
 
     var assert = chai.assert;
 
+    var skipTestIE = function() {
+        if (window.navigator.userAgent.indexOf('Windows') >= 0) {
```
Comment (ipeychev) [inline test/plugins/test/dragresize.js]: This condition doesn't skip the tests on IE, it skips them on Windows. Please use:
`if (CKEDITOR.env.ie) {`
to skip tests on all IE. You may also skip only some version of IE if the functionality doesn't work only on a particular version of IE.

**Comment by antoniopol06** [comment]
Hi @ipeychev !

Sorry for that.

I just changed those things. I will try to do that this does not happen in the future.

**Comment by ipeychev** [comment]
It is OK, Antonio. Thanks for the efforts to make the editor better.

**Comment by ipeychev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ipeychev** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/liferay/alloy-editor/compare/12f96cc~2...12f96cc) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**